app/build.gradle

```diff
@@ -50,4 +50,9 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
     // RX
     implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
+    // Recycler
+    implementation 'com.android.support:recyclerview-v7'
+    // Glide
+    implementation 'com.github.bumptech.glide:glide:4.11.0'
+    implementation 'com.google.android.material:material:1.3.0'
```

Comment:
you do not need this dependence because you already have it on the line 43

Reply:
Done thx